script.js

```diff
@@ -1,4 +1,336 @@
+var gameMode = "NA";
+var playerTurn = "NA";
+var sequenceToggle = "OFF";
+var player1 = 0;
+var player2 = 0;
+var player1Storage = 0;
+var player2Storage = 0;
+var player1Name = "Player1Name";
+var player2Name = "Player2Name";
+var dice1 = 0;
+var dice2 = 0;
+var dice3 = 0;
+var dice4 = 0;
+
+// Entry of Player's Name
+var enterPlayerName = function (input) {
+  playerName = input;
+  return playerName;
+};
+
+//Dice number generator
+var randomDiceRoll = function () {
+  var randomDiceRoll = Math.ceil(Math.random() * 6);
+  return randomDiceRoll;
+};
+
+//select sequence
+var selectSequence1 = function (dice1, dice2) {
+  var selectSequence1 = dice1 + "" + dice2;
+  return Number(selectSequence1);
+};
+var selectSequence2 = function (dice2, dice1) {
+  var selectSequence2 = dice2 + "" + dice1;
+  return Number(selectSequence2);
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // Enter Player 1 Name
+  if (player1Name == "Player1Name") {
+    var playerName = enterPlayerName(input);
+    player1Name = playerName;
+    return player1Name + " has joined the game";
+  }
+  // Enter Player 2 Name
+  if (player2Name == "Player2Name") {
+    var playerName = enterPlayerName(input);
+    player2Name = playerName;
+    return player2Name + " has joined the game";
+  }
+  // Input validation for game modes : Normal, Lowest Combined, Auto Generate
+  if (
+    input == "Normal" ||
+    input == "Lowest" ||
+    input == "Auto" ||
+    playerTurn == "End"
+  ) {
+    gameMode = input;
+    playerTurn = "Player 1";
+    return (myOutputValue = input + " has been selected. Lets begin!");
+  } else myOutputValue = "Choose only Normal, Lowest or Auto~";
+
+  if (gameMode == "Normal") {
+    if (input == "End") {
+      playerTurn = "End";
+      return (myOutputValue = "Whats your next game? Normal/Lowest/Auto");
+    }
+    // player1 turn
+    if (playerTurn == "Player 1") {
+      var diceRoll1 = randomDiceRoll();
+      var diceRoll2 = randomDiceRoll();
+      dice1 = diceRoll1; // storing player 1 numbers into global
+      dice2 = diceRoll2;
+      myOutputValue =
+        player1Name +
+        " has rolled " +
+        dice1 +
+        " and " +
+        dice2 +
+        ". <br> Select the sequence.";
+      sequenceToggle = "ON";
+      playerTurn = "Player 1 Sequence Toggle";
+      return myOutputValue;
+    }
```

Comment:
player 1 roll dice and player 2 roll dice are the same logic. You could refactor it into a function and reuse it instead of rewriting the same logic